### PR TITLE
Add Twitter social button

### LINKED ISSUE #3127: What about creating a Twitter account for Prettier (news, announcements, etc)
https://twitter.com/prettier (for some reason it's suspended, but I'm sure you know people that know people)

Plus it will be easier to praise Prettier on Twitter :)

### PR DESCRIPTION
**Pull request by lipis**
Related: #3127

  <a href="https://twitter.com/PrettierCode">
    Twitter Follow
  </a>

or

[Twitter Follow](https://twitter.com/PrettierCode)

---

Closes #3127 (edited by @ikatyang)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,9 @@
   <a href="#badge">
     <img alt="code style: prettier" src="https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square">
   </a>
+  <a href="#badge">
```
Comment (azz) [inline README.md]: Wrong link

File: README.md [hunk 2]
```diff
@@ -35,6 +35,9 @@
...
+    <img alt="Twitte Follow" src="https://img.shields.io/twitter/follow/prettiercode.svg?label=Follow+Prettier&style=flat-square">
```
Comment (azz) [inline README.md]: Typo

Comment (lipis) [reply README.md]: :D

File: README.md (hunk 1 above)
Comment (lipis) [reply README.md]: Done

**Comment by lipis** [comment]
1200+ followers.. not bad :)

**Comment by lipis** [comment]
maybe on both?

**Comment by lipis** [comment]
So here is how they look like.. 

screen shot 2017-11-22 at 11 31 32
screen shot 2017-11-22 at 11 32 09

**Comment by lipis** [comment]
What's the status with this one? I updated the README with lower case as everything else

**Comment by lipis** [comment]
We could possible also mention it in the up coming release notes about the Twitter account :) #3333

File: README.md [hunk 3]
```diff
@@ -35,6 +35,9 @@
   <a href="#badge">
     <img alt="code style: prettier" src="https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square">
   </a>
+  <a href="https://twitter.com/PrettierCode">
+    <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/prettiercode.svg?label=follow+prettier&style=flat-square">
```
Comment (suchipi) [inline README.md]: Could we change the alt text from "Twitter Follow" to "Follow Prettier on Twitter"? (And in website/core/Footer.js too)

Comment (lipis) [reply README.md]: Done!

**Comment by suchipi** [comment]
Sorry, I meant the img's `alt` attribute, not the label (the label was fine)

**Comment by lipis** [comment]
There you go

**Comment by lipis** [comment]
🍻

**Pull request merged**